.github/workflows/deploy.yml

```
@@ -0,0 +1,391 @@
+          curl -f https://staging.api.zippicks.com/wp-json/zippicks/v1/health || exit 1
+
+      - name: Notify Slack
+        if: always()
+        uses: 8398a7/action-slack@v3
+        with:
+          status: ${{ job.status }}
+          text: |
+            Staging deployment ${{ job.status }}
+            Commit: ${{ github.sha }}
+            Author: ${{ github.actor }}
+          webhook_url: ${{ secrets.SLACK_WEBHOOK }}
```

actionlint (1.7.7)

input "webhook_url" is not defined in action "8398a7/action-slack@v3". available inputs are "author_name", "channel", "custom_payload", "fields", "github_base_url", "github_token", "icon_emoji", "icon_url", "if_mention", "job_name", "mention", "status", "text", "username"

(action)